ENIGMAsystem/SHELL/Platforms/SDL/Event.cpp

```diff
@@ -211,9 +212,22 @@ void SDL_Event_Handler::windowResized(const SDL_Event *event) {
   enigma_user::draw_clear(enigma_user::window_get_color());
 }
 
-void SDL_Event_Handler::joyDeviceAdded(const SDL_Event *event) { addGamepad(event->cdevice.which); }
+void SDL_Event_Handler::joyDeviceAdded(const SDL_Event *event) {
```

Comment:
As a minimal addition to your pr it might be nice to print something  like "Joystick 3 ("DUAL SHOCK 3") connected/disconnected" in debug mode for these events